src/projects/version.md

```diff
@@ -1,7 +1,107 @@
-# Project Versioning
+## Project Versioning
 
-Radare2 have its own versioning control system. It's like a simple `git`. But also acts as a way to wrap the use of **git** under the same API.
+Radare2 includes a built-in version control system that operates similarly to Git, offering project versioning under the command `ravc2` (or its API counterpart, `rvc`). This allows users to manage their project history, track changes, revert to previous versions, and more. Radare2's project versioning system integrates seamlessly with Git, allowing you to choose between Radare2's versioning system or wrapping Git under the same interface.
```

Comment:
Still very unclear WHY would I choose `rvc` over `git`... totally understand why one would use version control, but it's not clear at all (to me) wht your custom version control system would be preferable against git.